docs/quick-start/README.md

```diff
@@ -3,8 +3,7 @@
 Mira is published as an NPM package. To use Mira, it must be added as a dependency to your own project. To get started, choose one of the following sample apps and generate your own project repository:
 
 - [mira-sample-s3-webhosting](https://github.com/nearform/mira-sample-s3-webhosting/generate)
-
-NB Just one sample app available at this time
+- [titus](https://github.com/nearform/titus)
```

Comment:
Looking at this now, I'm not sure if this format works for the Titus sample, or maybe for either - they will deviate. I'll come back to this point later.

Reply:
I'll open a new issue to deal with this - it is fine in this PR